main.go

```diff
@@ -1,21 +1,47 @@
 package main
 
 import (
+	"context"
 	"fmt"
+	"log"
 	"net/http"
+	"proxy/helper"
+	"proxy/lnd"
 
 	"github.com/gin-gonic/gin"
+	"github.com/lightningnetwork/lnd/lnrpc"
+)
+
+const (
```

Comment:
It would be better to read these 2 variables from environment variables instead of hardcoding them.